sources/assets/Stride.Core.Assets/StridePackagesToUpgrade.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Stride.Core.Assets;
+public static class StridePackagesToUpgrade
+{
+    public static string[] PackageNames =
```

Comment:
`Contains()` in a `List<T>` or `Array` is a *O(n)* operation. Better use a `HashSet<string>` for speed.

Reply:
And if possible, sort them by name to make it easier to maintain.

Reply:
good calls, changed to `HashSet` and alphabetized the list per region and cascading.